tests/integration/Application/CommandHandler/Entity/PublishEntityProductionCommandHandlerTest.php

```diff
@@ -108,14 +108,6 @@ public function test_it_can_publish()
         $manageEntity->shouldReceive('isManageEntity')->andReturnTrue();
         $manageEntity->shouldReceive('getEnvironment')->andReturn('production');
 
-        $this->publishEntityClient
-            ->shouldReceive('publish')
-            ->once()
-            ->with($manageEntity, $manageEntity)
-            ->andReturn([
-                'id' => '123',
-            ]);
-
         $manageEntity
             ->shouldReceive('getMetaData->getEntityId')
```

Comment:
If we keep using mockery, maybe we should consider the newer syntax
https://docs.mockery.io/en/latest/reference/alternative_should_receive_syntax.html

Reply:
That might be a good thing, but I'm not going to change it now.